modules/EasyAbp.PaymentService.WeChatPay/src/EasyAbp.PaymentService.WeChatPay.Domain/EasyAbp/PaymentService/WeChatPay/PaymentServiceWeChatPayHandler.cs

```diff
@@ -51,12 +55,13 @@ public virtual async Task HandleAsync(XmlDocument xmlDocument)
             payment.SetExternalTradingCode(reader["transaction_id"] ??
                                            throw new XmlDocumentMissingRequiredElementException("transaction_id"));
             
-            if (reader["result_code"] == "SUCCESS")
+            if (context.IsSuccess)
             {
                 payment.CompletePayment(_clock.Now);
             }
             else
             {
+                //TODO:Should log or return failed response?
```

Comment:
Information about the failure is logged into the RefundRecord entity, and the response should be a success because the WeChat only want to know if we got the notification.